doc/code_snippets/snippets/config/instances.enabled/memtx/config.yaml

```diff
@@ -0,0 +1,18 @@
+memtx:
+  memory: 100000000
```

Comment:
Nit: IMO, showing a larger value might look better. For example, the default value is ~ 268 mb and I hardly see the scenarios when a user needs a lower value. I'd set this value to 1 Gb.